### PR TITLE
Issue #1198 Fixed(Notification Interval Title Added)

### LINKED ISSUE #1198: Notification Mode Title not set - Minor Fix
**Actual Behaviour**
img2

**Expected Behaviour**
img1

**Steps to reproduce it**
Clearly Visible in Settings Tab

**Would you like to work on the issue?**
Yes. 
@mananwason @the-dagger Could I send a PR  for this?

### PR DESCRIPTION
**Pull request by shreyasnbhat**
Fixes #1198 

Changes: In the screenshots for the issue

Screenshots for the change: 
img1

### CODE REVIEW

**Comment by harshitagupta30** [comment]
@shreyasnbhat Can you please update your commit message according to [guidelines](https://github.com/fossasia/open-event-android/blob/development/docs/commitstyle.md) :+1:

File: android/app/src/main/res/xml/settings.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
         <ListPreference
             android:defaultValue="@string/default_mode_notification"
             android:dialogTitle="@string/settings_mode_notification"
+            android:title="Interval"
```
Comment (harshitagupta30) [inline android/app/src/main/res/xml/settings.xml]: Instead of hard coding "interval" please place it in strings.xml

**Comment by shreyasnbhat** [comment]
@harshitagupta30  Required Fixes done.

**Comment by mananwason** [comment]
@shreyasnbhat It's not really a bug. Please add a more descriptive message to your commit. Thanks!

**Pull request closed without merging**